release-notes/swan-lake-alpha4-release-note.md

```diff
@@ -118,3 +179,15 @@ To view bug fixes, see the GitHub milestone for Swan Lake <VERSION> of the repos
 #### Ballerina Packages Updates
 
 ### Breaking Changes
+
+- Unused variables declared with `var` of which the inferred type includes subtypes of `error` result in a compilation error.
```

Comment:
This is hard to understand. Is it correct to say the following? 
```suggestion
- A compilation error occurs if the inferred type of unused variables that are declared with `var` includes subtypes of `error`. 
```

Reply:
How about 
```suggestion
- A compilation error occurs if the inferred type of an unused variable that is declared with `var` includes a subtype of the `error` type.
```

Reply:
+1